src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/RazorProjectServiceTest.cs

```diff
@@ -956,9 +918,9 @@ public async Task UpdateDocument_ChangesDocumentInAllOwnerProjects()
         const string RootNamespace = "TestRootNamespace";
         const string DocumentFilePath = "C:/path/to/document.cshtml";
 
-        var ownerProjectKey1 = await _projectService.AddProjectAsync(
+        var ownerProjectKey1 = await _projectService.GetTestAccessor().AddProjectAsync(
```

Comment:
💭 should we just create a member for the test accessor?

Reply:
I'm not sure if I understood your suggestions, but IMO we should actively get away from having members on production code "for testing".

Reply:
I meant in the `RazorProjectServiceTest`. It already is using a member variable for `_projectService` so I just assumed (without reading the full class) that allocating a new `TestAccessor` is just waisted typing becuase the underlying object being tested will always be `_projectService`

Reply:
I kinda like how the methods being called to set up test data look distinctly different in the tests. I also don't care about saving allocations of test accessors, or anything else in tests really (which I know some people definitely don't agree with 😛)

Reply:
allocations aren't my problem. I'm lazy and like typing fewer things. It's already here though let's stick with it :)

Reply:
Laziness wins (by me not changing the code 😛)